src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,20 @@
+@import '../utils/mixins';
+
+.stars {
+  display: flex;
+  flex-direction: row;
+
+  &__star {
+    background-image: url(/src/images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+    width: 16px;
+    height: 16px;
+
+    &:nth-child(-n + 4),
+    &--4 {
+      background-image: url(/src/images/star-active.svg);
```

Comment:
The `&:nth-child(-n + 4)` selector seems to be incorrect for a BEM block as it applies styles based on the position of the element, which is not recommended for BEM methodology. BEM suggests using modifiers to handle state changes or variations of a block or element. If you want to have different states for stars, consider using modifiers like `&--active` and applying them to the necessary elements in the HTML.